src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
```

Comment:
You could pass a base URL for this controller, for example "/api/books"
This is done with the following tag:
@RequestMapping